gcovr/configuration.py

```diff
@@ -617,12 +617,19 @@ def merge_options_and_set_defaults(partial_namespaces, all_options=None):
         default=None,
     ),
     GcovrConfigOption(
-        "show_branch", ["-b", "--branches"], config='txt-branch',
+        "show_branch", ["-b", "--branches"],
```

Comment:
For the --branches flag, removing the `txt-branches` config key is a backwards-incompatible change. Existing configuration files would no longer work.

Reply:
Ok, but this key is also not only for txt output.

Reply:
I know, but we didn't think about that at the time when this config key was introduced :)

If we want to break backwards compatibility, then deliberately. We can do that if we really want to: the documentation on config files still says this feature is still experimental, after all.

Reply:
I've undone the change and we can create a seperate PR to cleanup the configuration keys.